CybridSDK/Utilities/Formatters/AssetFormatter.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  AssetFormatter.swift
+//  CybridSDK
+//
+//  Created by Erick Sanchez Perez on 27/03/23.
+//
+
+import Foundation
+import CybridApiBankSwift
+
+struct AssetFormatter {
+
+    /// Method to set amount in any asset in format to trade (the amount always handle in input)
+    /// Example:
+    /// - 10 USD --> 1000
+    /// - 10.5 USD --> 1050
+    /// - 0.064218090000000000 ETH --> 64218090000000000
+    /// Parameters:
+    ///  - asset: AssetBankModel to take the decimals and symbol ($)
+    ///  - amount: CDecimal object with preloaded amount
+    static func forTrade(_ asset: AssetBankModel, amount: CDecimal) -> String {
```

Comment:
We should choose a better name for this method as we'll use it in places other than just trade. How about 'fromInput?' or something like that.